### PR TITLE
[docs] Improving Gitsign docs 

### PR DESCRIPTION
**Pull request by erikaheidi**
#### Summary
This PR brings a few improvements to the Gitsign docs:

- Copy edits / improvements;
- A separate page with instructions on how to inspect signatures (gitsign/inspecting.md) - **needs review to make sure the information is accurate**
- Updated preview images replacing the default Nuxtjs ones - this should fix the preview that shows up whenever we share pages from the documentation.

```release-note
NONE
```

### CODE REVIEW

File: content/en/community.md [hunk 1]
```diff
@@ -15,8 +15,4 @@ Docs and other calendar invites may be shared directly with this group, so pleas
 
 ## Slack
 
-You can also keep in touch by joining our [Slack channel](https://sigstore.slack.com). Use [this invite link](https://links.sigstore.dev/slack-invite) to join.
-
-[//]: # (In case the invite link is expired, ping Dan on Slack of via Twitter: @lorenc_dan)
-
-_Having issues to join Slack? Ping [@lorenc_dan](https://twitter.com/lorenc_dan) on Twitter._
+You can also keep in touch by joining our [Slack channel](https://sigstore.slack.com). Use [this invite link](https://links.sigstore.dev/slack-invite) to join.
```
Comment (cpanato) [inline content/en/community.md:18]: This will be hosted under chainguard domain or under sigstore? If it is under chainguard i think we need to reword this to not say our slack and something as the sigstore slack ...

Comment (erikaheidi) [reply content/en/community.md:18]: This is on the Sigstore domain! The first version is up at docs.sigstore.dev/gitsign

Comment (ltagliaferri) [reply content/en/community.md:18]: Yeah this is under the Sigstore domain!

File: content/en/gitsign/inspecting.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+position: 155
+---
+
+Git commit signatures use [CMS/PKCS7 signatures](https://datatracker.ietf.org/doc/html/rfc5652). You can inspect the underlying data and certificate associated with a project's HEAD commit by running:
+
+```shell
+git cat-file commit HEAD | sed -n '/BEGIN/, /END/p' | sed 's/^ //g' | sed 's/gpgsig //g' | sed 's/SIGNED MESSAGE/PKCS7/g' | openssl pkcs7 -print -print_certs -text
+```
+
+You should see output similar to this:
```
Comment (ltagliaferri) [inline content/en/gitsign/inspecting.md:14]: I would make "see" read as "receive" or something like that for more inclusive language

File: content/en/gitsign/installation.md [hunk 3]
```diff
@@ -20,7 +20,16 @@ go install github.com/sigstore/gitsign@latest
 
 The resulting binary will be placed at `$GOPATH/bin/gitsign`. 
 
-### Installing Gitsign with the `.deb` package (Debian / Ubuntu Linux)
+### Installing Gitsign with Homebrew
+
+You can install Gitsign for MacOS systems using Homebrew. Use `brew tap` to add Sigstore's repository to your system, then run `brew install` to get it installed:
```
Comment (ltagliaferri) [inline content/en/gitsign/installation.md:25]: Homebrew works with Linux and the Windows Subsystem for Linux too, so I would probably write something like this (with link to Homebrew too)

```
If you are using [Homebrew](https://docs.brew.sh/) as a package manager for macOS or Linux, you can use it to install Gistign. Use `brew tap` to add Sigstore's repository to your system, then run `brew install` to get it installed:
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Minor suggestions, thank you for these updates and for updating the preview images too!

File: content/en/community.md (hunk 1 above)
Comment (cpanato) [reply content/en/community.md:18]: cool, i dont know maybe was my mistake, but i saw the page under the cg domain. thanks for the clarification and sorry for the mess

File: content/en/gitsign/inspecting.md (hunk 2 above)
Comment (erikaheidi) [reply content/en/gitsign/inspecting.md:14]: you're 1000% right, thank you! fixing it
